src/bidmanager.js

```diff
@@ -120,12 +120,6 @@ exports.addBidResponse = function (adUnitCode, bid) {
   if (bidsBackAll()) {
     this.executeCallback();
   }
-
-  if (bid.timeToRespond > $$PREBID_GLOBAL$$.bidderTimeout) {
```

Comment:
This code was modified and merged to master with [34e6874](https://github.com/prebid/Prebid.js/commit/34e6874455111658d8ba39173f6b80824c6a7452#diff-96cd0debf7b766d72d2bcf987385e960R127). Please update your branch with latest. This code is also the timeout callback trigger and we should prefer this approach to using Javascript `setTimeout` for reliability.